src/screens/contacts/contacts.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { View, Text, SafeAreaView, Button } from 'react-native';
+import { ListItem, Avatar, Overlay, Input } from 'react-native-elements';
+import { FlatList, ScrollView, TouchableOpacity } from 'react-native-gesture-handler';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import styles from './contacts.styles';
+
+import { _getUser } from '../../utils/asyncUtil';
+
+class ContactsScreen extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            allUsers: [],
+           
+            listUsers: []
+        }
+    }
+
+    async componentDidMount() {
+        let user = await _getUser()
+        this.convertUserObjectToArray(this.props.usersProfile, user);
+    }
+
+    convertUserObjectToArray = (usersObj = {}, own) => {
```

Comment:
Why this usersObj = {}.?

Reply:
to over come error "cannot find x of undefined"

Reply:
So, here if we read {} of something it will be undefined but let say if we don't initialize it ( By-default =>undefined) and undefined of something results into an error. Right @vikas-quant ?